### PR TITLE
Makes Ministation Toxin Airlocks Heat-Proof (V2)

### LINKED ISSUE #14139: Heat superconducts through the Toxins airlocks on Ministation.

### PR DESCRIPTION
**Pull request by swankcookie**
Fixes #14139
:cl: Swankcookie
tweak: After numerous disintegration-related deaths, Nanotrensen has gotten around to replacing the seals around the the Toxins Airlocks. Heat will now stay where it is supposed too.
/:cl:

### CODE REVIEW

File: cfg/admin.txt [hunk 1]
```diff
@@ -13,3 +13,4 @@ giacomand
 rockdtben
 MrStonedOne
 Jordie0608
+Swankcookie
```
Comment (Jordie0608) [inline cfg/admin.txt]: I think you've got the wrong file here; this is for enabling access to client-side debugging and profiling tools.
/config/admins.txt is what's used for the admin tools.

Comment (swankcookie) [inline cfg/admin.txt]: yeah, I know, i accidentally created this branch from the first branch i made, when i was trying to figure out how to selfmin. I'm already in the other one.  I can remove myself from this one if you want, but i was told it didn't matter, anyway.

Comment (Aranclanos) [inline cfg/admin.txt]: REMOVE THYSELF

Comment (swankcookie) [inline cfg/admin.txt]: consider mineself removed

**Pull request merged**